api-scanner/src/test/resources/kotlin-vararg-method/kotlin/net/corda/example/KotlinVarargMethod.kt

```diff
@@ -1,5 +1,5 @@
 package net.corda.example
 
 interface KotlinVarargMethod {
-    fun action(vararg arg: Any?)
+    fun action(vararg arg: Array<String>)
```

Comment:
I'm not sure this makes sense - your `action` function is asking for a variable number of `String` **arrays**! Did you mean:
```kotlin
fun action(vararg arg: String)
```
Which means that the API expectation would be:
```java
public abstract void action(String...)
```
I think our test cases should be realistic. If you _want_ a pathological case (which wouldn't be unreasonable) then that should be in addition to the realistic case.

Reply:
No I did mean to do that - your suggestion in one of the earlier comments was to do a `.replace("[]", "...")` and I hypothesised that if someone had a vararg list of lists parameter you would end up with `String......` instead of `String[]...`. But I agree it is... out there. I'll add the more basic case back in.